tests/app/test_model.py

```diff
@@ -642,7 +667,11 @@ def test_create_a_new_brief_response_with_data(self):
             try:
                 brief_response.validate()
             except ValidationError as v:
-                errors = v.message
+                try:
+                    errors = v.message
+                except AttributeError:
+                    (errors,) = v.args
+
                 assert 'essentialRequirements' not in errors
```

Comment:
should void unnecessarily throwing exception. without line 670 to line 674, line 675 should be 'assert 'essentialRequirements' != str(v)

Reply:
This looks fine to me for now.  We can think about changing it in the future.